android/build.gradle

```diff
@@ -1,44 +1,119 @@
 buildscript {
-    ext.safeExtGet = {prop, fallback ->
-        rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
-    }
-    // The Android Gradle plugin is only required when opening the android folder stand-alone.
-    // This avoids unnecessary downloads and potential conflicts when the library is included as a
-    // module dependency in an application project.
-    if (project == rootProject) {
```

Comment:
@avishayil 

why you remove these lines? the issue https://github.com/avishayil/react-native-restart/pull/106 has come back by removing the lines!

can you please revert this?